docs/hooks/medication-refill.md

```diff
@@ -0,0 +1,174 @@
+# `medication-refill`
+
+| Metadata | Value
+| ---- | ----
+| specificationVersion | 1.0
+| hookVersion | 0.1.0
+
+## Workflow
+
+The `medication-refill` hook fires when a medication refill is requested for an existing prescription of a specific medication. A refill request may be made as part of an encounter or out-of-band through a pharmacy. There may not be either an encounter or user in context when the refill request is received.  A CDS service may use this hook to deliver medication refill protocol guidance to a clinician.
+
+## Context
+
+The set of medications proposed or in progress of being prescribed. All FHIR resources in this context MUST be based on the same FHIR version. All FHIR resources in the medications object MUST have a status of _draft_.
+
+Field | Optionality | Prefetch Token | Type | Description
+----- | -------- | ---- | ---- | ----
+`userId` | OPTIONAL | Yes | *string* | The id of the current user.<br />For this hook, the user is expected to be of type [Practitioner](https://www.hl7.org/fhir/practitioner.html).<br />For example, `Practitioner/123`
+`patientId` | REQUIRED | Yes | *string* |  The FHIR `Patient.id` of the current patient in context
+`encounterId` | OPTIONAL | Yes | *string* |  The FHIR `Encounter.id` of the current encounter in context
+`medications` | REQUIRED | No | *object* | DSTU2 - FHIR Bundle of _draft_ MedicationOrder resources <br/> STU3 - FHIR Bundle of _draft_ MedicationRequest resources
+
+### Example (STU3)
```

Comment:
Add R4 example

Reply:
@brynrhodes In the R4 example, the MedicationRequest has:
                  "status":"draft",
                  "intent":"order",

I was curious as to the status to expect in a MedicationRequest for an order generated by a Refill Authorization Request (RAR): 
http://community.fhir.org/t/medicationrequest-status-and-refill-authorization-requests/1628 

Lloyd responded to the post indicating he thought that a new RAR would either be a Task resource or if it were a MedicationRequest resource then it wold be a MedicationRequest resource with: 
   "status":"active" 
   "intent":"proposal". 
Is Lloyd's response compelling enough to update the example? Alternatively, is this something that ought to be taken to the pharmacy work group for discussion?